libc/cmake/modules/LLVMLibCCompileOptionRules.cmake

```diff
@@ -60,7 +62,25 @@ function(_get_common_compile_options output_var flags)
       list(APPEND compile_options "-ffixed-point")
     endif()
 
-    list(APPEND compile_options "-fno-builtin")
+    # Builtin recognition causes issues when trying to implement the builtin
+    # functions themselves. The GPU backends do not use libcalls so we disable
+    # the known problematic ones. This allows inlining during LTO linking.
+    if(LIBC_TARGET_OS_IS_GPU)
+      list(APPEND compile_options "-fno-builtin-bcmp")
```

Comment:
Can you do this all in one big list(APPEND) call

Reply:
I think something like this might be better on second thought.
```
foreach(fn strlen memmem ...)
  ...
endforeach()
```

Reply:
The one list not embedded in the foreach line would be most readable

Reply:
Okay, I made it a separate list. Should be easier to add to now if needed.